pkg/networkmanager/wireguard.jsx

```diff
@@ -160,7 +166,11 @@ export function WireGuardDialog({ settings, connection, dev }) {
                         throw cockpit.format(_("Peer #$0 has invalid endpoint port. Port must be a number."), index + 1);
                     }
                 }
-                return ({ ...peer, allowedIps: peer.allowedIps.trim().split(',') });
+                if (peer.persistentKeepalive.trim()) {
+                    if (isNaN(Number(peer.persistentKeepalive)))
+                        throw cockpit.format(_("Peer #$0 has invalid persistent keepalive. It must be a number."), index + 1);
```

Comment:
Probably it should also check for ≥ 1 and possibly some maximum? Also, can this be a float?

Reply:
The dbus type for Keepalive is 'u', so it has to be >=0. Passing negative or float shows an error in the UI. Like below:

![image](https://github.com/cockpit-project/cockpit/assets/108616679/6c599c7f-556d-40c3-aa77-233dbb2a7da2)

I opened an issue for a better error message at #19645.

0 (zero) means "not set" as usual.

I couldn't find a documented maximum value, but little experiment shows it can be in the range 0 to 65535 (i.e. 2^16 possible values).. after that it overflows without any error.